service/src/test/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderServiceTest.java

```diff
@@ -682,15 +625,15 @@ void shouldUpdateExistingOrdersBundle() {
             service.updateCase(eventData, hearings, emptyList(), ordersBundles);
 
             HearingOrdersBundle expectedOrdersBundle = originalOrdersBundle.toBuilder()
-                .orders(newArrayList(cmoOrder, hearingOrder1, hearingOrder3))
+                .orders(newArrayList(cmoOrder, hearingOrder1, hearingOrder2, hearingOrder1, hearingOrder3))
```

Comment:
please confirm if there should be two`hearingOrder1`

Reply:
Yeah should be both